File: API/Entities/Education.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class Education
    {
        public int Id { get; set; }
        public required  string Degree { get; set; }
        public required  string Institution { get; set; }
        public required  string Location { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public ICollection<string> Achievements { get; set; } = new List<string>();
    }
}

File: topcv/src/models/Education.ts
export interface Education {
    id: number;                      // Unique identifier for the education
    degree: string;                  // Degree obtained
    institution: string;             // Institution name
    location: string;                // Location of the institution
    startDate: string;               // Start date of the education (ISO format)
    endDate: string;                 // End date of the education (ISO format)
    achievements: string[];          // List of achievements
}
